### DISCUSSION TITLE
Histogram x-axis limits configuration

### DISCUSSION THREAD

**Discussion by jmbelanger** [fbdesignpro/sweetviz#92] [Ideas]
Hi,

    I just ran a quick test with sweetviz  on a csv file (dataframe) of mine with 20 columns.
Very neat and useful. 
I was wondering if it possible to specify somewhere (in a configuration file ... I noticed a "sweetviz_defaults.ini" ) 
the limits (min, max,  and interval... why not ?)  of the x-axis  so that the 20 histograms are plotted 
with same limits, in order  to ease the visual comparison ?

Just a tought.

Thanks,

Jean-Marc 

| Jean-Marc Bélanger, M.Sc.
| Senior Physical Scientist
| Environmental Numerical Prediction Research 
| Environment Canada and Climate Change
| E-mail: Jean-Marc.Belanger@canada.ca

**Comment by fbdesignpro** [comment]
Hi @jmbelanger! Thank you for the kind words. This does makes a lot of sense and sounds like something that shouldn't be too much work. My schedule to work on sweetviz has been fluctuating wildly so it's a bit hard to have an ETA on implementation but it's definitely going to be towards the top of the stack as far as new features. 

I will definitely post here once it is implemented.

**Reply by jmbelanger** [reply]
Ok, thanks for the quick reply!
Cheers,
Jean-Marc

**Comment by bd-didier-morel** [comment]
Hello François, first many thanks for your outstanding work with SweetViz, it's certainly the EDA reporting package I'm using the most since I found it. Just one question, maybe specific to how I use it (from R via reticulate). Missing values for character variables are treated like one distinct value, hence appearing as NA in the barplots and counted in the number of distinct values but leaving a Missing count without any value (--). Is it by design? Is there a way to have missing values counted as missing and not displayed in the barplots? Thanks a lot for your help. Best regards, Didier.
